### ISSUE TITLE
When running the stats task in a huge project it crashes because "Too many open files"

### ISSUE THREAD

**Issue by mgttlinger** [orrsella/sbt-stats#10]
I suspect that the opened files are not closed properly after inspection by the task as the configured limit of 1637049 simultaneously opened files feels like it should suffice when processed files are closed afterwards.

**Comment by orrsella** [comment]
Do you get any stack trace or anything else when the plugin crashes? Where do you see the "Too many open files" message?

**Comment by mgttlinger** [comment]
Yes, the message is from the stack trace. Will run again and attatch the trace when I'm back at work.
